hack/test-cmd.sh

```diff
@@ -443,14 +443,15 @@ osc delete all -l app=guestbook
 echo "edit: ok"
 
 osc delete all --all
-osc new-app https://github.com/openshift/ruby-hello-world -l app=ruby
```

Comment:
> why no new-app?

in v1beta1, the imagestreamtag would be created for the image that new-app wanted.  In v1beta3, that does not happen until a build completes.  Turns out that v1beta1 was incorrect and the v1beta3 behavior is correct, but test-cmd.sh doesn't necessarily have working docker, so you can't reliably have a build run.  Because I still wanted to test `scale`, I created a deploymentconfig.